docs/netlify.toml

```diff
@@ -1197,6 +1197,12 @@
   to = "/quickstart/428201/custom-function"
   status = 301
 
+[[redirects]]
+  # redirect 
+  from = "/configuration/custom-registry"
+  to = "/configuration/engine#configuring-custom-registries"
+  status = 301
+
```

Comment:
If deleting the caching page as suggested, we should add a redirect for that as well.